consensus/polybft/state.go

```diff
@@ -471,8 +461,8 @@ func (s *State) insertStateSyncEvent(event *StateSyncEvent) error {
 
 // getStateSyncEventsForCommitment returns state sync events for commitment
 // if there is an event with index that can not be found in db in given range, an error is returned
-func (s *State) getStateSyncEventsForCommitment(fromIndex, toIndex uint64) ([]*StateSyncEvent, error) {
-	var events []*StateSyncEvent
+func (s *State) getStateSyncEventsForCommitment(fromIndex, toIndex uint64) ([]*types.StateSyncEvent, error) {
+	var events []*types.StateSyncEvent
 
 	err := s.db.Update(func(tx *bolt.Tx) error {
```

Comment:
Shouldn't we use `View` function here instead, since we are basically reading the data from the DB?

```suggestion
	err := s.db.View(func(tx *bolt.Tx) error {
```

Reply:
Weird it was like that before 😛